homeassistant/components/sensor/bitcoin.py

```diff
@@ -79,19 +79,19 @@
     'miners_revenue_usd': ['Miners revenue', 'USD'],
     'btc_mined': ['Mined', 'BTC'],
     'trade_volume_usd': ['Trade volume', 'USD'],
-    'difficulty': ['Difficulty', ''],
+    'difficulty': ['Difficulty', ' '],
```

Comment:
These  changes shouldn't be in here. I will take a look today at making the badges hold more info